packages/grid/_modules_/grid/components/column-header-item.tsx

```diff
@@ -92,8 +90,8 @@ export const ColumnHeaderItem = React.memo(
           />
         )}
         <ColumnHeaderSeparator
-          resizable={!disableColumnResize && column.resizable}
-          onResize={handleResize}
+          resizable={disableColumnResize ? false : Boolean(column.resizable)}
```

Comment:
please refactor to 
`resizable={!disableColumnResize || !!column.resizable }`

Reply:
Ok, I find it easier to read with ternaries and explicit boolean casting but no strong preferences.

Reply:
I would encourage using the suggesting feature, it reduces potential confusion:

```suggestion
          resizable={!disableColumnResize || !!column.resizable}
```

Reply:
Actually, the proposed boolean logic doesn't work.